src/js/controller/controller.js

```diff
@@ -281,6 +282,28 @@ define([
                 }
             }
 
+            function autostartFallbackOnItemReady() {
+                cancelAutostartFallbackOnItemReady();
+                checkAutoStartLastContext = { bail: false };
+                _model.once('itemReady', checkAutoStartLast, checkAutoStartLastContext);
+            }
+
+            function cancelAutostartFallbackOnItemReady() {
+                checkAutoStartLastContext.bail = true;
+                _model.off('itemReady', checkAutoStartLast);
+            }
+
+            function checkAutoStartLast() {
+                // Use promise as setImmediate() to allow synchonous calls to load() and play() set the playReason
+                Promise.resolve().then(() => {
```

Comment:
Interesting. I'd long wondered if there was a browser equivalent of `setImmediate`.  Seems only IE/Edge will ever implement it.